examples/md-flexible/src/configuration/MDFlexConfig.h

```diff
@@ -340,6 +340,12 @@ class MDFlexConfig {
   MDFlexOption<std::array<double, 3>, 0> boxMax{
       {1, 1, 1}, "box-max", true, "Upper back right corner of the simulation box."};
 
+  /**
+   * loadBalancingInterval
+   */
+  MDFlexOption<unsigned int, 0> loadBalancingInterval{
```

Comment:
```suggestion
  MDFlexOption<unsigned int, __LINE__> loadBalancingInterval{
```
This is super important, otherwise, you break the CLI parser. The number here is the identifier token for `getOpt`.